### PR TITLE
Add domain type filter

### LINKED ISSUE #1805: add domain type filter
please add the domain type as one of the selector conditions

### PR DESCRIPTION
**Pull request by qinyong666**
// Describe your PR here; eg. Fixes #1805

add domain type filter

### CODE REVIEW

**Review by Asxing — DISMISSED** [review]

**Comment by yu199195** [comment]
please add sql in shenyu-admin

**Comment by qinyong666** [comment]
> please add sql in shenyu-admin

I have added

File: shenyu-admin/src/main/resources/META-INF/schema.sql [hunk 1]
```diff
@@ -298,6 +298,7 @@ INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_v
...
 INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_value`, `desc`, `sort`, `enabled`, `date_created`, `date_updated`) VALUES ('50', 'automaticTransitionFromOpenToHalfOpenEnabled', 'AUTOMATIC_HALF_OPEN', 'open', 'true', '', 1, 1, '2021-07-18 22:59:17', '2021-07-18 22:59:17');
 INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_value`, `desc`, `sort`, `enabled`, `date_created`, `date_updated`) VALUES ('51', 'automaticTransitionFromOpenToHalfOpenEnabled', 'AUTOMATIC_HALF_OPEN', 'close', 'false', '', 2, 1, '2021-07-18 22:59:34', '2021-07-18 22:59:34');
+INSERT IGNORE INTO `shenyu_dict` (`id`, `type`,`dict_code`, `dict_name`, `dict_value`, `desc`, `sort`, `enabled`, `date_created`, `date_updated`) VALUES ('52', 'paramType', 'PARAM_TYPE', 'domain', 'domain', 'domain', 8, 1, '2021-05-30 19:40:41', '2021-05-30 20:15:23');
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/META-INF/schema.sql:301]: please also add in h2.sql

Comment (qinyong666) [reply shenyu-admin/src/main/resources/META-INF/schema.sql:301]: > please also add in h2.sql

have added

**Review by yu199195 — APPROVED** [review]

**Pull request merged**